src/EditorFeatures/CSharpTest/KeywordHighlighting/DiscardParameterHighlighterTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.CSharp.Highlighting.KeywordHighlighters;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.KeywordHighlighting
+{
+    public class DiscardParameterHighlighterTests : AbstractCSharpKeywordHighlighterTests
+    {
+        internal override IHighlighter CreateHighlighter()
+        {
+            return new DiscardParameterHighlighter();
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordHighlighting)]
+        public async Task TestDiscardWithoutType()
+        {
+            await TestAsync(
+@"using System;
+
+class DiscardExample
+{
+    void Method()
+    {
+        int i = int.TryParse("""", out {|Cursor:[|_|]|});
+    }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordHighlighting)]
+        public async Task TestDiscardWithVarType()
+        {
+            await TestAsync(
+@"using System;
+
+class DiscardExample
+{
+    void Method()
+    {
+        int i = int.TryParse("""", out var {|Cursor:[|_|]|});
```

Comment:
💡 Should add a test showing that this does not highlight the argument if you write it as `@_`. This token has the same `ValueText` as `_`, but different `Text`, and in this case the difference is meaningful.

Reply:
Added tests covering that case.